### PR TITLE
Photo credit attribution

### PR DESCRIPTION
**Pull request by sillypears**
Allow attribution to the photo if it was not from the maker

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -223,6 +228,11 @@ function gDriveParse (catalog, tabs) {
             giveaway = true
             text = text.replace(regGiveaway, '')
           }
+          console.log(photoCreditMatch)
```
Comment (zekth) [inline src/utils.js:231]: Could you remove that statement?

File: templates/README.md [hunk 2]
```diff
@@ -34,6 +34,8 @@ To manipulate some attributes of the catalogs you can add those to the gdoc:
 - **Sculpt Header** `(ka_cast_xxx)` Specify how the cap is made. Availabe values: `resin`/`mixed`
 - **Colorway Header** `(count #)` will add the total count to the colorway page. Value should be an integer: `(count 1)`, `(count 13)`
 - **Colorway Header** `(*)` will add a "Commission" tag to the colorway page. `(*)` is removed from the colorway title.
+- **Colorway Header** `(giveaway)` or `(give-away)` to note if a colorway was specificly for a giveaway
```
Comment (zekth) [inline templates/README.md:37]: Good catch

**Comment by zekth** [comment]
I dont get why the lint is failing. Need to investigate. Did you run this locally?

**Comment by sillypears** [comment]
Yes, ran it locally to verify that the photoCredit field was being populated.
image

What I realized was the change in how gdocs were being pulled. I was seeing this during both linting and building, which was fixed when I moved from `G_API_KEY` to providing the cert. I didn't look at the linting jobs to see if they needed to be updated as well or not, but that was the error message I was seeing.

**Pull request merged**